content/process/repositories.md

```diff
@@ -116,6 +116,16 @@ Reviewers should only block a PR by `Requesting Changes` if one or more elements
 
 Once a PR has been reviewed and approved by at least one reviewer, it can be merged. The only way we allow merging is by squashing all the PR's commits into a single one, as this keeps the `master` branch and the changelog clean and easy to follow.
 
+!!! warning "Closing Pull Requests"
+
+    When closing a Pull Request, make sure to always specify the reason for closing it by adding a comment to the PR while closing it.
+    This will ensure that collaborators can follow the process and understand the reasons for the PR being closed, and ensures that the repository has a well-documented history.
```

Comment:
```suggestion
    This will help the collaborators follow the process and understand the reasons for closing the PR, as well as ensuring that the repository has a well-documented history.
```
Not sure if my suggestion is the best possible grammatically; the reason I made it is to avoid the repeating of "ensure" word in the same sentence.

Reply:
Ok